reana_workflow_controller/rest.py

```diff
@@ -927,3 +934,179 @@ def get_workflow_status(workflow_id):  # noqa
+          description: >-
+            Request succeeded. Info about workflow, including the status is
+            returned.
+          schema:
+            type: object
+            properties:
+              message:
+                type: string
+              workflow_id:
+                type: string
+              organization:
+                type: string
+              status:
+                type: string
+              user:
+                type: string
+          examples:
+            application/json:
+              {
+                "message": "Workflow successfully launched",
+                "workflow_id": "256b25f4-4cfb-4684-b7a8-73872ef455a1",
+                "organization": "default_org",
+                "status": "running",
+                "user": "00000000-0000-0000-0000-000000000000"
+              }
+        400:
+          description: >-
+            Request failed. The incoming data specification seems malformed.
+          examples:
+            application/json:
+              {
+                "message": "Malformed request."
+              }
+        403:
+          description: >-
+            Request failed. User is not allowed to access workflow.
+          examples:
+            application/json:
+              {
+                "message": "User 00000000-0000-0000-0000-000000000000
+                            is not allowed to access workflow
+                            256b25f4-4cfb-4684-b7a8-73872ef455a1"
+              }
+        404:
+          description: >-
+            Request failed. Either User or Workflow doesn't exist.
+          examples:
+            application/json:
+              {
+                "message": "User 00000000-0000-0000-0000-000000000000 doesn't
+                            exist"
+              }
+            application/json:
+              {
+                "message": "Workflow 256b25f4-4cfb-4684-b7a8-73872ef455a1
+                            doesn't exist"
+              }
+        500:
+          description: >-
+            Request failed. Internal controller error.
+    """
+
+    try:
+        organization = request.args['organization']
+        user_uuid = request.args['user']
+        workflow = Workflow.query.filter(Workflow.id_ == workflow_id).first()
+        status = request.json
+        if not (status in STATUSES):
+            return jsonify({'message': 'Status {0} is not one of: {1}'.
+                            format(status, ", ".join(STATUSES))}), 400
+        if not workflow:
+            return jsonify({'message': 'Workflow {} does not exist'.
+                            format(workflow_id)}), 404
+        if not str(workflow.owner_id) == user_uuid:
+            return jsonify(
+                {'message': 'User {} is not allowed to access workflow {}'
+                 .format(user_uuid, workflow_id)}), 403
+        if status == START:
+            return start_workflow(organization, user_uuid, workflow)
+        elif status == STOP:
```

Comment:
This is not implemented yet. I would remove it since changing the status in DB won't do anything, if the workflow is running it will keep running anyways and user will see `finished` status while it is not true.

Reply:
I created it to test status changes similar to `get_workflow_status` tests. If `stop` is not on your roadmap in the nearest future, I will remove it.

Reply:
Indeed it is in the roadmap, but still not implemented so this "changing in the DB directly" would be kind of a "lie" because it is not how it will be implemented most probably.

If you want to check status changes you can get the status right after creating the workflow, which should be `created`, and then submit status change for `started` so you check if it works.